package.json

```
@@ -2,7 +2,7 @@
   "author": "Transcend Inc.",
   "name": "@transcend-io/airgap.js-types",
   "description": "TypeScript types for airgap.js interoperability with custom consent UIs",
-  "version": "8.29.3",
+  "version": "8.29.4",
```

I think this should be a minor version bump because it changes the interface
```suggestion
  "version": "8.30.0",
```